cli/dcoscli/package/main.py

```diff
@@ -682,7 +682,15 @@ def _bundle_uris(uris_directory, zip_file):
     :rtype: None
     """
 
-    for filename in sorted(os.listdir(uris_directory)):
+    uris = sorted(os.listdir(uris_directory))
+    # these uris will be found through a mustache template from the property
+    # name so make sure each name is unique.
+    uri_names = set(name.replace('.', '-') for name in uris)
+    if len(uris) != len(uri_names):
+        raise DCOSException(
+            'Error bundling package. Multiple assets map to the same property '
+            'name (periods [.] are replaced with dashes [-]): {}'.format(uris))
```

Comment:
@tamarrow `...format(uris)` list all of the packages not just the ones that collide.